input/pages/introduction.md

```diff
@@ -19,7 +19,7 @@ Except where noted, material from the base FHIR specification, and in particular
 
 As features and functionality are identified by this implementation guide that apply more broadly, those features may be promoted to the base FHIR specification.
 
-There is a new term of digital quality measure or dQM that is currently being defined across many interested parties. See the glossary definition: [https://hl7.org/fhir/us/cqfmeasures/2024May/glossary.html#digital-quality-measure-dqm] For the purposes of this IG, eCQM and dQM are synonymous. The focus of this IG is a  computable representation of a quality measure using HL7 FHIR and CQL.
+There is a new term of digital quality measure or dQM that is currently being defined across many interested parties (see the definition in [Glossary](https://build.fhir.org/ig/HL7/cqf-measures/glossary.html)).
```

Comment:
1. the link added is CI Build, please correct.
2. Add back the sentence: 
"For the purposes of this IG, eCQM and dQM are synonymous. The focus of this IG is a computable representation of a quality measure using HL7 FHIR and CQL."